crowbar_framework/app/views/barclamp/dns/_edit_attributes.html.haml

```diff
@@ -3,7 +3,12 @@
     = header show_raw_deployment?, true
 
   .panel-body
-    = string_field :domain
+
+    %fieldset
+      = string_field :domain, :disabled => "disabled"
```

Comment:
I think this is set using yast when you first install crowbar, so crowbar comes up with this proposal already activated and the user would never see the barclamp UI before it's active. Maybe someone else can confirm how it works?